### ISSUE TITLE
BatchNorm

### ISSUE THREAD

**Issue by vinay-ram1999** [vinay-ram1999/TradeForecast#7]
Apply batch norm after pooling layer in ConvLSTM model and test.
